core/sequencer/sequencer.go

```diff
@@ -120,6 +130,7 @@ func (s *Sequencer) ListenForNewDomains(ctx context.Context, refresh time.Durati
 				return fmt.Errorf("admin.List(): %v", err)
 			}
 			for _, d := range domains {
+				knownDomains.Set(1.0, d.DomainID)
```

Comment:
does the Gauge metric only permit floats to be stored? not ints?

Reply:
That's right.  However I can use `1` rather than `1.0` because, in Go, numbers are not typed.